docusaurus.config.js

```diff
@@ -209,6 +209,37 @@ module.exports = {
         ignorePatterns: ['**/docs/js/**']
       }
     ],
+    [
+      '@docusaurus/plugin-client-redirects',
+      {
+        redirects: [
+          {
+            from: '/docs/java/guides/ai-core-deployment',
+            to: '/docs/java/ai-core/ai-core-deployment'
```

Comment:
I guess in JS, we would call this AI API right? @jjtang1985

Reply:
yes

Reply:
@ZhongpinWang can I get an approval on this PR if JavaScript will have the same sidebar as Java?